# Catch exception

* try
* catch

{% embed include file="src/examples/dart-intro/catch_exception.dart" %}
{% embed include file="src/examples/dart-intro/catch_exception.out)


